astropy/time/tests/test_basic.py

```diff
@@ -258,6 +258,17 @@ def test_precision(self):
         assert t.iso == '2010-01-01 00:00:00.000000000'
         assert t.tai.utc.iso == '2010-01-01 00:00:00.000000000'
 
+
+    def test_precision_input(self):
+        """Verifies that precision can only be 0-9 (inclusive). Any other
+        value should raise a ValueError exception."""
+        with pytest.raises(Exception) as e:
```

Comment:
In both of these you should use `with pytest.raises(ValueError, match='precision attribute must be an int'):`.